e2etest/bgp_test.go

```diff
@@ -394,6 +355,101 @@ var _ = ginkgo.Describe("BGP", func() {
 			}),
 		)
 	})
+
+	ginkgo.Context("BFD", func() {
+		table.DescribeTable("should work with the given bfd profile", func(bfd bfdProfile) {
+			configData := configFile{
+				Pools: []addressPool{
+					{
+						Name:     "bgp-test",
+						Protocol: BGP,
+						Addresses: []string{
+							"192.168.10.0/24",
+							"fc00:f853:0ccd:e799::/124",
+						},
+					},
+				},
+				Peers:       withBFD(peersForContainers(frrContainers), bfd.Name),
+				BFDProfiles: []bfdProfile{bfd},
+			}
+			err := updateConfigMap(cs, configData)
+			framework.ExpectNoError(err)
+
+			for _, c := range frrContainers {
+				pairExternalFRRWithNodes(cs, c, func(container *frrcontainer.FRR) {
+					container.NeighborConfig.BFDEnabled = true
+				})
+			}
+
```

Comment:
No need to call `validateFRRPeeredWithNodes` for BFD?

Reply:
I think I forgot about it